File: src/main/java/com/github/kuro46/embedscript/script/ScriptUtil.java
package com.github.kuro46.embedscript.script;

import java.util.StringJoiner;

public final class ScriptUtil {
    private ScriptUtil() {
    }

    public static String toString(Iterable<String> values) {
        StringJoiner joiner = new StringJoiner("][", "[", "]");
        for (String value : values) {
            joiner.add(value);
        }
        return joiner.toString();
    }

    public static String toString(String value) {
        return '[' + value + ']';
    }
}

File: src/main/kotlin/com/github/kuro46/embedscript/script/ScriptUtil.kt
package com.github.kuro46.embedscript.script

/**
 * @author shirokuro
 */
object ScriptUtil {

    fun toString(values: Iterable<String>): String {
        return values.joinToString("][", "[", "]")
    }

    fun toString(value: String): String {
        return "[$value]"
    }
}
